io_u.c

```diff
@@ -1808,6 +1808,8 @@ static void account_io_completion(struct thread_data *td, struct io_u *io_u,
 	if (td->parent)
 		td = td->parent;
 
+	td->priority_bit = io_u->priority_bit;
```

Comment:
I don't like tracking this in the td, that seems very random. Why track something that belongs to a specific IO in the thread data?

Reply:
I agree, it seems like this is unnecessary and could be more cleanly handled. I'll investigate a way to fix this.